docs/ContribOperators.md

```diff
@@ -1192,7 +1192,7 @@ This version of the operator has been available since version 1 of the 'com.micr
 <dt><tt>present_value</tt> (optional) : T</dt>
 <dd>present state for value with shape (batch_size, num_heads, total_sequence_length, head_size). If past_present_share_buffer is set, its shape is (batch_size, num_heads, max_sequence_length, head_size), while effective_seq_length = (past_sequence_length + kv_sequence_length).</dd>
 <dt><tt>qk</tt> (optional) : V</dt>
-<dd>normalized Q * K, of shape (batch_size, num_heads, 1, head_size). </dd>
+<dd>normalized Q * K, of shape (batch_size, num_heads, 1, total_sequence_length). </dd>
```

Comment:
normalized Q * K has shape (batch_size, num_heads, sequence_length, total_sequence_length)?

Reply:
Yes, I think it should be `(B, N, S, H) x (B, N, T, H)'  -> (B, N, S, T)`.

Reply:
oh, it is fine since sequence_length==1 for this op.